sdk/go/auto/cmd_test.go

```diff
@@ -309,11 +309,11 @@ func TestRunCanceled(t *testing.T) {
 	stdout, _, code, err := cmd.Run(ctx, e.CWD, nil, nil, nil, env, "preview", "-s", stackName)
 	if runtime.GOOS == "windows" {
 		require.ErrorContains(t, err, "exit status 0xffffffff")
-		require.Contains(t, stdout, "error: preview canceled")
+		require.Contains(t, stdout, ".*error: preview canceled.*|.*signal: interrupt", stdout)
```

Comment:
Should this also be `require.Regexp`? If it's the same check between both Windows and other platforms, should we just have one check above the `if runtime.GOOS == "windows" {` line?

Reply:
Indeed it should be.  Just a single check for this seems like a good idea, did that now.